.github/workflows/bpf-checks.yaml

```diff
@@ -7,6 +7,17 @@ on:
       - master
 
 jobs:
+    checkpatch:
+        name: checkpatch
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
```

Comment:
Did you find this makes a sensible difference? Should we use it in other GitHub workflows (not in this PR of course)?

Reply:
Lower perf when checking out, but required because I run checkpatch on each individual commit from the PR. From what I understand, without it, checking out the repo is done in such a way it contains only one commit. Other checks as far as I can tell don't work on the individual commits so no need to change.